packages/manager/apps/container/src/container/legacy/account-sidebar/UsefulLinks/constants.ts

```diff
@@ -30,6 +32,8 @@ const consts: UsefulLinkConstants = {
       TN: `${helpRoot}/fr-tn${homeIndex}`,
     },
     tasks: 'https://www.status-ovhcloud.com/',
+    supportTicketUrl: 'https://help.ovhcloud.com/csm?id=contact_us_ovh',
+    viewSupportTicketsUrl: 'https://help.ovhcloud.com/csm?id=csm_cases_requests',
```

Comment:
**nitpick:** No need to add both. 
/dedicated/support/tickets/new -> This is the URL for Support page in EU and CA
/dedicated/ticket -> This is the URL for Support ticket page in US

So, the above links must be added based on the region.